main/getting-started/ertp-introduction.md

```diff
@@ -72,29 +72,29 @@ We refer to
 that kind as a `mint`'s *Brand*. So if a `mint` issues Quatloos, it's a
 Quatloo `brand` `mint`.  Only `mints` can issue new digital assets. To mint
 new assets of a particular kind, you must have a reference to that
-kind's `mint`. 
+kind's `mint`.
 
 - **Issuers**: Create empty `purses` and manipulate and operate on
 `payments`. `Issuers` verify and move digital assets and are
-the authority on which `payments` and `purses` hold what digital assets.
+the authority on contents of `payments` and `purses` using their brand.
 
 An `issuer`'s special admin facet is a `Mint`, and that `Mint` and `Issuer`
 have a one-to-one relationship. With a reference to an `Issuer`, you can
 check the validity of a `payment` in that `issuer`'s assets;
 i.e. If you have a reference to the Quatloos `issuer`, you can validate
-any `payment` made in Quatloos. You can also claim the `payment` either 
+any `payment` made in Quatloos. You can also claim the `payment` either
 as a new `payment` to yourself or a `purse` you control.
 
-`Issuers` should be gotten from a trusted source
+`Issuers` should be retrieved from a trusted source
```

Comment:
```suggestion
`Issuers` should be obtained from a trusted source
```
No objective best answer at this level of vagueness, but how does "obtained" sound? The term "retrieved" sounds like it was the response to some kind of retrieval request initiated by this party, which is too specific.